### PR TITLE
Python Kafka Consumption Example

### LINKED ISSUE #1: Python Kafka consumption examples
To progress the visualisation tool @AnthonyLim23 is working on, some demos of consuming the Kafka data in Python would be useful.

Initially, could you add a demo that reads from an event topic using [this](https://github.com/STFC-ICD-Research-and-Design/supermusr-data-pipeline/blob/main/schemas/dev1_digitizer_event_v1.fbs) schema and prints the metadata and event list sizes to the console.

### PR DESCRIPTION
**Pull request by Henry-Denny**
Closes #1.

### CODE REVIEW

File: consumer.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    )
+    print("Running:\t\t\t", metadata.Running())
+    print("Veto Flags:\t\t\t", metadata.VetoFlags())
+    print("Frame Number:\t\t\t", metadata.FrameNumber())
+    print("Period Number:\t\t\t", metadata.PeriodNumber())
+    print("Protons Per Pulse:\t\t", metadata.ProtonsPerPulse())
+    print("-------------------------------------------------------------\n")
+
+
+def listen(trace_topic: str):
```
Comment (DanNixon) [inline consumer.py]: Where should the broker address be set?

File: consumer.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    msg (ConsumerRecord): The message to be parsed.
+
+    Returns:
+    DigitizerEventListMessage: The deserialised payload.
+    """
+    payload = DigitizerEventListMessage.GetRootAs(msg.value, 0)
+    return payload
+
+
+def display_payload_information(payload: DigitizerEventListMessage):
```
Comment (DanNixon) [inline consumer.py]: Where are the actual events? This just prints the metadata.

File: schemas/aev1_frame_assembled_event_v1.fbs [hunk 3]
```diff
@@ -0,0 +1,13 @@
+include "frame_metadata_v1.fbs";
```
Comment (DanNixon) [inline schemas/aev1_frame_assembled_event_v1.fbs]: Do not include the schemas here, the single source of truth for those is https://github.com/STFC-ICD-Research-and-Design/supermusr-data-pipeline

**Review by DanNixon — CHANGES_REQUESTED** [review]
Looks good, just a few comments.

This repo will have more than just this code in it eventually, so can this be put in a new directory. Also include a README.md explaining what this is.

Could some form of dependency management also be used (whatever the default is for Python now).

**Review by DanNixon — CHANGES_REQUESTED** [review]
Could the `demos` directory be renamed something more descriptive (reflecting what the example actually does).

Not sure the `__init__.py` at the repository root is needed.

There is no actual event information printed, could a summary of the events in the message be added?

**Comment by Henry-Denny** [comment]
I'm not sure exactly what you mean by the event information. Would that be the `Voltage` and `Channel` fields?

**Comment by DanNixon** [comment]
> I'm not sure exactly what you mean by the event information. Would that be the `Voltage` and `Channel` fields?

Yes, but just print the number of events and first 5 or so elements since these contain a lot of data.

**Review by DanNixon — APPROVED** [review]

**Pull request merged**